cfgov/jinja2/v1/_includes/molecules/_vars-content.html

```diff
@@ -0,0 +1,9 @@
+{% macro _privacy_statement() %}
```

Comment:
As discussed in chat, possibly move this to be a private helper macro within `email-signup-form.html`.